docker/.env

```diff
@@ -6,8 +6,21 @@ MF_NGINX_SSL_PORT=443
 MF_NGINX_MQTT_PORT=1883
 MF_NGINX_MQTTS_PORT=8883
 
-## NATS
-MF_NATS_URL=nats://nats:4222
+# Message Broker
+MF_BROKER_TYPE=nats
+
+## Nats
+MF_NATS_PORT=4222
+MF_NATS_URL=nats://broker:4222
+
+## RabbitMQ
+MF_RABBITMQ_PORT=5672
+MF_RABBITMQ_HTTP_PORT=15672
+MF_RABBITMQ_USER=mainflux
+MF_RABBITMQ_PASS=mainflux
+MF_RABBITMQ_COOKIE=mainflux
+MF_RABBITMQ_VHOST=/
+MF_RABBITMQ_URL=amqp://mainflux:mainflux@broker:5672/
```

Comment:
You can reuse previous values to construct the URL: `MF_RABBITMQ_URL=amqp://$MF_RABBITMQ_USER:$MF_RABBITMQ_PASS@broker:$MF_RABBITMQ_PORT$MF_MF_RABBITMQ_VHOST`